openwisp_controller/config/migrations/0056_ow_zt-to-global.py

```diff
@@ -0,0 +1,18 @@
+from django.db import migrations
+import json
+
+def change_owzt_to_global(apps, schema_editor):
+    Template = apps.get_model('config', 'Template')
+    for template in Template.objects.all():
```

Comment:
I'd filter for VPN client templates, eg (writing from the top of my head without testing, please double check):

```suggestion
    for template in Template.objects.filter(type='vpn', vpn__backend='openwisp_controller.vpn_backends.ZeroTier' ):
```

Reply:
you are right on the dot